squarelet/users/viewsets.py

```diff
@@ -110,6 +112,23 @@ def get_object(self):
             return super().get_object()
 
 
+class PressPassUserMembershipViewSet(
+    mixins.ListModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = Membership.objects.none()
+    serializer_class = PressPassUserMembershipsSerializer
+    permission_classes = (DjangoObjectPermissionsOrAnonReadOnly,)
```

Comment:
Why `DjangoObjectPermissionsOrAnonReadOnly` instead of `DjangoObjectPermissions` - we generally require logging in to view user details